### DISCUSSION TITLE
Maintainer and/or status of this project

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [containers/podman-compose#827] [Q&A]
Hi @muayyad-alsadi  and other members.  This repository has not had much action over the past many months.  We have had other folks ask if they could become a maintainer of the project.  Is that something you're interested in?  @muayyad-alsadi are you still interested in being the owner/maintainer of this project?

**Comment by fccagou** [comment]
First of all, thanks a lot for the works done to build this project.

I'm new user of  podman-compose and this discussion is important to know the status of the project _(owners ?, maintainers ? , Redhat Supports, active pull request validation, ...)_  and determine the way I can use it and/or participate to it. 

Thanks.

**Comment by az-z** [comment]
you might want to read @rhatdan comments in #276 . Albeit they were made 3 years ago. 
I can't seem to locate more recent statements from RH about podman future and docker/podman eco system. Specifically migration/adoption of "compose" spec in rootless environment.

**Reply by rhatdan** [reply]
Red Hat does not support podman-compose formally, but we are fine if people use it, just like any open source projects.
